mne/tests/test_source_estimate.py

```diff
@@ -619,6 +619,14 @@ def test_stc_methods():
         assert_array_almost_equal(stc_new.data, stc.data, 5)
 
 
+def test_stc_resamp_noop():
```

Comment:
This will fix one of the CI failures
```suggestion
@testing.requires_testing_data
def test_stc_resamp_noop():
```